sydent/http/servlets/emailservlet.py

```diff
@@ -25,16 +28,19 @@
     SessionExpiredException,
 )
 
+if TYPE_CHECKING:
+    from sydent.sydent import Sydent
+
 
 class EmailRequestCodeServlet(Resource):
     isLeaf = True
 
-    def __init__(self, syd, require_auth=False):
+    def __init__(self, syd: "Sydent", require_auth: bool = False) -> None:
         self.sydent = syd
         self.require_auth = require_auth
 
     @jsonwrap
-    def render_POST(self, request):
+    def render_POST(self, request: Request) -> Dict:
```

Comment:
Maybe?

```suggestion
    def render_POST(self, request: Request) -> Dict[str, Any]:
```

You'll need to import `Any` separately.

There's a few places we could do something similar to this in this PR.